platform-metrics/src/main/java/org/hypertrace/core/serviceframework/metrics/PlatformMetricsRegistry.java

```diff
@@ -142,6 +152,42 @@ private static void initTestingMetricsReporter() {
     METER_REGISTRY.add(new SimpleMeterRegistry());
   }
 
+  private static void initPrometheusPushGatewayReporter(String serviceName,
+      int reportIntervalSec,
+      String gatewayUrlAddress) {
```

Comment:
We are interchangeably using `pushUrlAddress` and `gatewayUrlAddress` for variable names/consts. Can we be consistent everywhere by calling it 'pushGatewayUrlAddress`?

Reply:
I was thinking pushUrlAddress in config shouldn't contain PushGateway because it's not limited to Push Gateway. And Http Address would do. 

I can make the this one to be pushUrlAddress too.